src/main/java/com/tileman/TilemanProfileManager.java

```diff
@@ -88,22 +88,22 @@ private TilemanGameRules loadGameRules(TilemanProfile profile) {
     }
 
     private void saveGameRules(TilemanProfile profile, TilemanGameRules rules) {
-        if (profile.getAccountHash() == -1) {
+        if (profile == TilemanProfile.NONE) {
```

Comment:
Should this be object comparison? Or implement hashcode/equals on TilemanProfile and use `.equals`

Reply:
Good point. It was working, but flimsier that way. I've gone ahead and done as suggested.

Reply:
Very clean